cmd/rm-main.go

```diff
@@ -308,7 +308,8 @@ func removeSingle(url, versionID string, opts removeOpts) error {
 		case http.StatusBadRequest, http.StatusMethodNotAllowed:
 			ignoreStatError = true
 		default:
-			ignoreStatError = st == http.StatusServiceUnavailable && opts.isForce && opts.isForceDel
+			_, ok := pErr.ToGoError().(ObjectMissing)
+			ignoreStatError = st == http.StatusServiceUnavailable || ok || st == http.StatusNotFound && opts.isForce && opts.isForceDel
```

Comment:
Aren't some parenthesis missing here ?

> ( http.StatusServiceUnavailable || ok || st == http.StatusNotFound ) && opts.isForce && opts.isForceDel

We don't have to ignore url2Stat() when --purge is not provided

Reply:
done and fixed @vadmeste